app/Http/Controllers/Table/PortNacController.php

```diff
@@ -88,6 +95,9 @@ public function formatItem($nac)
         $item['mac_oui'] = Rewrite::readableOUI($item['mac_address']);
         $item['mac_address'] = Rewrite::readableMac($item['mac_address']);
         $item['port'] = null; //free some unused data to be sent to the browser
+        if (isset($item['device_id'])) {
```

Comment:
This is always true.

Reply:
seems that you are right. don't know how I ended up convinced I needed to add it ...

Reply:
done